PublishWorkflow/PublishWorkflowChecker.php

```diff
@@ -166,4 +163,32 @@ public function isGranted($attributes, $object = null)
 
         return $this->accessDecisionManager->decide($token, $attributes, $object);
     }
+
+    private function getTokenStorage()
+    {
+        if (null === $this->tokenStorage) {
+            if ($this->container->has('security.token_storage')) {
+                $this->tokenStorage = $this->container->get('security.token_storage');
+            } else {
```

Comment:
this leads to a test failure because the checker would now explode in a situation where no security is avialble: 

1) Symfony\Cmf\Bundle\CoreBundle\Tests\Unit\PublishWorkflow\PublishWorkflowCheckerTest::testNoSecurityContext

Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: You have requested a non-existent service "Service not defined".